lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -269,7 +269,13 @@ def _set_sgi_ip_addr(self, if_name: str):
                 self._restart_dhclient(if_name)
             else:
                 # for system port, use networking config
-                if_up_cmd = ["ifup", if_name]
+                try:
```

Comment:
I know this is not really related to this PR, but can you add check for static IP and see if there is need to set it before flushing existing config.
I don't want to reset interface config if it not required.

Reply:
This part of the code being called only in case, when there is no static IP configuration. This means, that the user desired to use the "default" configuration with the DHCP server. This might lead to two different cases:
1) device already used DHCP server and there is no need to reset any configuration;
2) device has an "old" static IP assigned and the DHCP client killed.

For case 1, we can add a condition to check whether `dhclient` is running for the particular interface and skip all the interface reset routine. 
Case 2 will always require the interface reset to give the possibility for the system to run `dhclient` correctly.

Did I get your idea correctly?

Reply:
@berezovskyi-oleksandr we shld also cover static Ip config case, where static IP config is same as current IP we can skip the flush and set operation.

Reply:
@pshelar , setting static IP doesn't require any flushes currently. It flushes and re-configures the interface only in case of empty static IP configuration while in non-NAT mode.

Setting static IP code flow involves only killing the dhclient and setting the desired static IP.

Reply:
I am referring to ip flush operation on line no 294.
It would be better if we get rid of all static IP configuration if expected IP is already set on interface.

Reply:
@pshelar got your idea. Implemented